examples/deploy.py

```diff
@@ -18,7 +18,7 @@
     try:
         print('Deploying ubuntu')
         application = await model.deploy(
-            'ubuntu-10',
+            'ubuntu',
```

Comment:
I liked having the int in the string, just to demonstrate that you can deploy specific versions of things.

If this was breaking things, though, I understand taking it out ...